lib/active_model_serializers/rspec/matchers/serializer.rb

```diff
@@ -0,0 +1,59 @@
+require 'active_model_serializers/test/serializer'
+
+module ActiveModelSerializers
+  module RSpec
+    module Matchers
+      module Serializer
+        extend ActiveModelSerializers::Test::Serializer
+        extend ActiveSupport::Concern
+
+        included do
+          setup :setup_serialization_subscriptions
+          teardown :teardown_serialization_subscriptions
+
+          ::RSpec::Matchers.define :use_serializer do |expected|
+            match do
+              serializer_matches?(expected)
+            end
+            failure_message do
+              message
+            end
+          end
+        end
+
+        def serializer_matches?(expectation)
+          @assert_serializer.expectation = expectation
+          @assert_serializer.message = message
+          @assert_serializer.response = response
+          @assert_serializer.matches?
+        end
+
+        def message
+          @assert_serializer.message
+        end
+
+        class Base < ActiveModelSerializers::Test::Serializer::AssertSerializer
+          def subscribe
```

Comment:
What do you mean by this?
- I called the class `Base` for the lack of a better name. It's already inside `ActiveModelSerializers::RSpec::Matchers::Serializer` so it doesn't need any more specification.
- I'm changing the `subscribe` method for my own, so it works with `CollectionSerializer`s.